### DISCUSSION TITLE
Multiple tenants per user

### DISCUSSION THREAD

**Discussion by 8byr0** [django-tenants/django-tenants#725] [Q&A]
Hello!
From what I've seen so far, users are tenant specific (stored in tenant schema).

What would be the best approach to allow users to belong to multiple tenants ?

Thanks for advises !

**Comment by iamunadike** [comment]
Its not possible for user to belong to multiple tenants, because the schemas represent apartment, if you intend for an django app of users to be shared features, then configure your apps to be in SHARED_APPS tuple 
Then you can create an algorithm to match specifics for a tenant user accessing that feature which is now shared
Shared apps live mostly in the Postgresql Public Schema

**Reply by tomturner** [reply]
I always point people to https://github.com/Corvia/django-tenant-users when they ask this question
